src/signal.rs

```diff
@@ -1351,3 +1351,34 @@ extern "C" fn tree_view_column_trampoline(this: *mut GtkTreeViewColumn,
         f: &Box<Fn(TreeViewColumn) + 'static>) {
     f(TreeViewColumn::wrap_pointer(this))
 }
+
+#[cfg(gtk_3_16)]
```

Comment:
Due to the need for `cfg`s it's probably better to put this whole addition into a submodule like done above.